lib/trie/nodes.js

```diff
@@ -504,6 +528,10 @@ class Hash extends Node {
     return this;
   }
 
+  [inspect]() {
+    return `<Hash: ${this.data.toString('hex')}>`;
+  }
+
```

Comment:
Why not remove the old `inspect()` functions as well?

Reply:
To be backwards compatible with Node.js versions prior to 6.4.0